backend/game/cards/keep_card.py

```diff
@@ -0,0 +1,11 @@
+from abc import abstractmethod
+from game.cards.card import Card
+
+
+class KeepCard(Card):
+    def immediate_effect(self, player_that_bought_the_card, other_players):
+        player_that_bought_the_card.add_card(self)
+
+    @abstractmethod
+    def special_effect(self, player_that_bought_the_card, other_players):
```

Comment:
I think this should be called maybe like passive effect, but the issue i can see here is that it can also be determined by different things (such as dice rolls, etc) takes in more than just the player who bough card and other players